### PR TITLE
switch to versioneer

### PR DESCRIPTION
**Pull request by roxyboy**
I changed the version control to versioneer as suggested by #65.

### CODE REVIEW

**Comment by rabernat** [comment]
I'm happy to see you working on this.

If you'd like feedback from the other devs, you might consider
- adding descriptions to your PRs
- waiting at least a day before merging

**Comment by roxyboy** [comment]
The code coverage decreased due to `_version.py`. It comes from the versioneer package but is that something I should also have tests for @rabernat ?

**Comment by rabernat** [comment]
Consider excluding it from coverage, as done here:

https://github.com/xgcm/xgcm/blob/master/.coveragerc

File: setup.py [hunk 1]
```diff
@@ -1,29 +1,74 @@
 import os
+import versioneer
 from setuptools import setup, find_packages
 PACKAGES = find_packages()
 
-# Get version and release info, which is all stored in xrft/version.py
-ver_file = os.path.join('xrft', 'version.py')
-with open(ver_file) as f:
-    exec(f.read())
+VERSION = 'v0.1'
```
Comment (rabernat) [inline setup.py]: Might want to remove this `VERSION` variable, since it is now unused.

File: setup.py [hunk 2]
```diff
@@ -1,29 +1,74 @@
...
+#             author_email=AUTHOR_EMAIL,
+#             platforms=PLATFORMS,
+#             version=VERSION,
+#             packages=PACKAGES,
+#             package_data=PACKAGE_DATA,
+#             requires=REQUIRES)
+#
+#
+# if __name__ == '__main__':
+#     setup(**opts)
```
Comment (rabernat) [inline setup.py]: Remove all these comments please.

**Comment by rabernat** [comment]
What version is versioneer giving you now that this has been implemented?

```
import xrft
print(xrft.__version__)
```

**Comment by roxyboy** [comment]
It gives `0.1-beta+76.g9bbebbc.dirty`

File: setup.py (hunk 2 above)
Comment (roxyboy) [reply setup.py]: Both changes are implemented.

**Pull request merged**